source/AI.cpp

```diff
@@ -2694,7 +2753,7 @@ bool AI::DoHarvesting(Ship &ship, Command &command)
 			// Only pick up flotsam that is nearby and that you are facing toward.
 			Point p = it->Position() - ship.Position();
 			double range = p.Length();
-			if(range > 800. || (range > 100. && p.Unit().Dot(ship.Facing().Unit()) < .9))
+			if(range > 800. || (range > 100. && p.Unit().Dot(ship.Facing().Unit()) < .9 && !ship.IsYours()))
```

Comment:
```suggestion
			// Player ships do not have a restricted field of view so that they target flotsam behind them.
			if(range > 800. || (range > 100. && p.Unit().Dot(ship.Facing().Unit()) < .9 && !ship.IsYours()))
```

Reply:
For more context, here's the original play tester report https://github.com/endless-sky/endless-sky/pull/6669#issuecomment-1126729227